app/bot/openai.go

```diff
@@ -26,6 +26,8 @@ type OpenAI struct {
 	prompt    string
 	superUser SuperUser
 
+	history *LimitedMessageHistory
```

Comment:
why is pointer here? all receivers for this struct already pointers, so unless I missed the goal it can be just a normal variable